### ISSUE TITLE
[CONTENT] Slide pages does not seem to be in order

### ISSUE THREAD

**Issue by yilmazdurmaz** [hedyorg/hedy#4480]
**Describe the issue**
What level, language and adventure does this concern?

I am making translations to the slides right now. I open Weblate and Hedy slides side by side, and pages don't seem to follow a meaningful order. (actual pages, no Weblate strings)

**Add a screenshot (optional)**
Make a picture or screenshot to show the issue. Tip! You can make a screenshot and simply paste the image into GitHub with ctrl-v.

**Expected text**
What should we write instead?

Slide pages should be in a topic order.

**Comment by Felienne** [comment]
Thanks for letting us know! I am afraid I do not yet understand this issue well enough to take action.

If you go to https://hedy.org/slides are the slides then in a wrong order?

Or does the text in weblate not follow the order as it shows on the website?

Thanks for extra info you can offer

**Comment by yilmazdurmaz** [comment]
- chapter are in order, so no problem there (though they could use naming more like "level 0")
- Weblate strings do not have to be in an order as they can be sorted later with scripts.

The thing is, "Welcome to level X" is a puchline for the "first" page, but it is not in many of them.

level 0 should be welcoming to Hedy in the first page then tell about programming
level 1 has 2 pages at the start that should be in the middle
level 2 slides starts with 2 pages that must come at the end
level 3 starts fine but "random" page comes before introducing lists

In short, page orders of Slides are broken. I have a long way on the translations, thus, for the moment, I stopped paying attention on this order anymore and moved on, so I can't way about later level slides.

I had started only yesterday and worked my way first on components having less total strings, and so after a few hundreds of others, only about a third of slides I could translate for the moment. But it seems this problem may happen in later level slides.

**Comment by Felienne** [comment]
Hi @yilmazdurmaz, sorry to press this but I am still unsure if we are talking about Weblate or not (I think it is about Weblate, but I am not sure) Could you add a screenshot of the issue so I can dive in?

**Comment by yilmazdurmaz** [comment]
How do you expect me to screenshot all the pages?

if the page you are showing is really the "3rd", the it is broken in the "Turkish" pages, because that is the first page I see.

**Comment by Felienne** [comment]
One screenshot with the URL would very much help, you don't have to do them all.

**Comment by Felienne** [comment]
Ah I see a broken page now!! Thanks for giving me the extra info that you were looking at Turkish!

<img width="585" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/a1f21b6b-8d61-4551-8e51-2cc7325b0044">

I will dive in, no more further info is needed from you, thanks a lot for letting me know.

**Comment by Felienne** [comment]
> @Felienne , I did not think this would be language based problem. this broken order happens in at least first 4 levels.

Yes, I see what the issue is, thanks again for sharing. You can continue translating if you want, I will fix the slides order as soon as I can.

> By the way, Weblate is currently in lockdown due to an "ES" language merge conflict. If you have experience with that, please work on it first please.

Should be fixed!

**Comment by Felienne** [comment]
Closing this because we are fixing the yamls in a larger way, see #4541
